workspaces/sandbox/plugins/sandbox/src/components/SandboxCatalog/SandboxCatalogBanner.tsx

```diff
@@ -108,7 +110,10 @@ export const SandboxCatalogBanner: React.FC = () => {
                       },
                     }}
                   >
-                    Welcome, {userData?.name}
+                    Welcome,{' '}
+                    {userData.givenName !== ''
+                      ? userData.givenName
+                      : userData.compliantUsername}
```

Comment:
```suggestion
                    Welcome, {userData.givenName || userData.compliantUsername}
```
Shorter and cleaner syntax

Reply:
makes sense ! thanks !